extensions/sql-database-projects/src/models/tree/fileFolderTreeItem.ts

```diff
@@ -29,7 +29,7 @@ export class FolderNode extends BaseProjectTreeItem {
 	}
 
 	public get treeItem(): vscode.TreeItem {
-		const folderItem = new vscode.TreeItem(this.uri, vscode.TreeItemCollapsibleState.Collapsed);
+		const folderItem = new vscode.TreeItem(this.fileSystemUri, vscode.TreeItemCollapsibleState.Collapsed);
```

Comment:
Just a question here: just updating the uri to fileSystemUri updates the git status. How?

Reply:
+1 to this question.  I'm guessing VS Code automatically checks git status for tree entries that have filesystem URIs (rather than just arbitrary paths), which would be some unexpected but welcome magic.

Reply:
yup the git status shows if the resourceUri of the vscode.TreeItem is set to the full path